sql-ballerina/tests/call-procedures-test.bal

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+string proceduresDb = "procedures";
+string proceduresDB = urlPrefix + "9002/procedures";
+
+
+type StringDataForCall record {
+    string varchar_type;
+    string charmax_type;
+    string char_type;
+    string charactermax_type;
+    string character_type;
+    string nvarcharmax_type;
+};
+ 
+type StringDataSingle record {
+    string varchar_type;
+};
+
+
+
+@test:BeforeGroups {
+	value: ["procedures"]	
+} 
+function initproceduresContainer() {
+	initializeDockerContainer("sql-procedures", "procedures", "9002", "procedures", "call-procedures-test-data.sql");
+
+}
+
+@test:AfterGroups {
+	value: ["procedures"]	
+} 
+function cleanproceduresContainer() {
+	cleanDockerContainer("sql-procedures");
+}
+
+
+
+@test:Config {
+    groups: ["procedures"],
+    dependsOn: ["testCreateProcedures1"]
+}
+function testCallWithStringTypes() returns @tainted record {}|error? {
+    int id = 2;
+    MockClient dbClient = checkpanic new (url = proceduresDB, user = user, password = password);
+    ProcedureCallResult ret = checkpanic dbClient->call(`call InsertStringData(${id},'test1', 'test2     ', 'c', 'test3', 'd', 'test4');`);
```

Comment:
This can be moved to a common function

Sample https://github.com/ballerina-platform/module-ballerina-sql/blob/master/sql-ballerina/tests/batch-execute-query-test.bal#L105

Reply:
Ok I will do it.

Reply:
Cannot use a common function for testCallWithStringTypes, testCallWithStringTypesInParams test cases, because they need client object for client->query method. For other testcases we can use a common function.

Reply:
I add common function for other testcases. Commit id is ```76d0e08c47bcd7120a490db97680c560671e913a```